scripts/deploy.sh

```diff
@@ -147,6 +161,14 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   fi
   rm -f /etc/openstack_deploy/conf.d/aodh.yml
   rm -f /etc/openstack_deploy/conf.d/ceilometer.yml
+  rm -f /etc/openstack_deploy/conf.d/gnocchi.yml
+
+  # Per https://github.com/rcbops/u-suk-dev/issues/749, compute hosts
+  # will not be considered to be part of the neutron_l3_agent and
+  # neutron_metadata_agent groups, so we are removing in the nova.yml
+  # environment file.
```

Comment:
I'd be -1 to this approach, as you're editing in-tree environment. It was explicitly moved in-tree to avoid deployers doing this.

The recommended action is to write an override file to /etc/openstack_deploy/env.d/nova.yml.